docs/design_proposals/kaniko-logs-leakage.md

```diff
@@ -0,0 +1,74 @@
+# Skaffold logs improvements
+
+* Author(s): Prashant Arya (@prary)
+* Design Shepherd: 
```

Comment:
Tejal Desai

Reply:
Feel free to add me here! i am interested in this discussion!

Reply:
sure